### PR TITLE
 Fix EEFNameSpacedPropertyEditorDialog issues in Mediators

### PR DESCRIPTION
**Pull request by dilee**
## Purpose
Fixing EEFNameSpacedPropertyEditorDialog issues in:
- CallMediator
- Enrich Mediator
- CalloutMediator
- FastXSLTMediator

### CODE REVIEW

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/CallMediatorPropertiesEditionPartForm.java [hunk 1]
```diff
@@ -921,35 +924,43 @@ protected Composite createEndpointXPathWidget(FormToolkit widgetFactory, final C
...
         EditingUtils.setID(endpointXPathText, EsbViewsRepository.AggregateMediator.OnComplete.aggregationExpression);
         EditingUtils.setEEFtype(endpointXPathText, "eef::Text");
         Control endpointXPathHelp =FormUtils.createHelpButton(widgetFactory, parent, propertiesEditionComponent.getHelpContent(EsbViewsRepository.CallMediator.Properties.endpointXpath, EsbViewsRepository.FORM_KIND), null); //$NON-NLS-1$
         endpointXpathElements = new Control[] {endpointXPathLabel, endpointXPathText, endpointXPathHelp};
         return parent;
     }
 	
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(), SWT.NULL, endpointXPath);
+        //valueExpression.setPropertyValue(valueExpressionText.getText());
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/CallMediatorPropertiesEditionPartForm.java]: Remove  the commented code.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/EnrichMediatorPropertiesEditionPartForm.java [hunk 2]
```diff
@@ -1553,42 +1549,48 @@ protected Composite createtargetXPathWidget(FormToolkit widgetFactory, final Com
...
-        // aggregationExpressionElements = new Control[] { aggregationExpressionLabel, aggregationExpressionText,
-        // aggregationExpressionHelp };
         targetXPathElements = new Control[] { targetXPathLabel, targetXPathText, targetXPathHelp };
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, targetXPath);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/EnrichMediatorPropertiesEditionPartForm.java]: Remove commented code.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FastXSLTMediatorPropertiesEditionPartForm.java [hunk 3]
```diff
@@ -812,6 +810,19 @@ public void focusGained(FocusEvent e) {
         dynamicSchemaKeyElements = new Control[] { dynamicSchemaKeyLabel, dynamicSchemaKeyText, dynamicSchemaKeyHelp };
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, fastXsltDynamicSchemaKey);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```
Comment (prabushi) [inline plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FastXSLTMediatorPropertiesEditionPartForm.java]: Remove the commented line.

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/CallMediatorPropertiesEditionPartForm.java (hunk 1 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/CallMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/8bf90d13170529d3776b1699ed2b01a4b8b110d9

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/EnrichMediatorPropertiesEditionPartForm.java (hunk 2 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/EnrichMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/8bf90d13170529d3776b1699ed2b01a4b8b110d9

File: plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FastXSLTMediatorPropertiesEditionPartForm.java (hunk 3 above)
Comment (dilee) [reply plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/FastXSLTMediatorPropertiesEditionPartForm.java]: Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/8bf90d13170529d3776b1699ed2b01a4b8b110d9

**Pull request merged**